ANGEL_URREGO_1011081750/Tarea-4-Clase-5/primo.js

```diff
@@ -0,0 +1,18 @@
+let number = Number(prompt("Ingrese un numero"));
+let esPrimo = true
+
+if(number <= 1){
+    esPrimo = false
+}else{
+    for(i = 2; i < number / 2; i++);
```

Comment:
aqui no entiendo muy bien que quisiste hacer

Reply:
Buenas noches profe, el if indica que si el número es menor que 1 o igual es false osea que cuando el número sea 1 o menor va a imprimir no es primo ya en el, else esta el for que realiza las divisiones desde el numero dos hasta que consiga una division que su residuo sea 0

Reply:
pero esa sintaxis te funciona? es decir el for (...lo que sea); y cerrando con punto y coma?